.gitignore

```diff
@@ -77,6 +77,10 @@ t/fuzzing/__pycache__/
 boofuzz-results/
 *.pyc
 *.wasm
+grpc_server_example
```

Comment:
We should not add directory grpc_server_example into gitignore?

Reply:
It's not directory, just grpc_server_example binary executable file
<img width="419" alt="image" src="https://user-images.githubusercontent.com/9354193/178229117-0663eab5-30a4-4de0-937f-ffa5ee12f62d.png">

Reply:
It will affect any new file in this directory?

Reply:
sorry, i have updated the .gitignore file, please check.